files/usr/share/cinnamon/applets/sound@cinnamon.org/applet.js

```diff
@@ -965,6 +966,9 @@ class CinnamonSoundApplet extends Applet.TextIconApplet {
         this._volumeMax = this._sound_settings.get_int(MAXIMUM_VOLUME_KEY) / 100 * this._control.get_vol_max_norm();
         this._volumeNorm = this._control.get_vol_max_norm();
 
+        this._mediakeysStep_settings = new Gio.Settings({ schema_id: VOLUME_STEP_DIR });
+        this._mediakeysStep = this._mediakeysStep_settings.get_int(VOLUME_STEP_KEY) / 100;
+
```

Comment:
Add a listener similar to: https://github.com/linuxmint/cinnamon/blob/6560b3be37f783a4c015254a8c9b84e1ff8457ab/files/usr/share/cinnamon/applets/sound%40cinnamon.org/applet.js#L1034

You could even use the same callback, just tack on something to update `this._mediakeysStep`